docs/testplan/pfc/PFC_PAUSE_RESPONSE_HEADROOM_README.md

```diff
@@ -0,0 +1,48 @@
+This document describes how to test the effect of PFC response times (a delay in response to pfc frames) from the IXIA on the headroom of the SONiC DUT for lossless traffic. For the purposes of our test, the data has been collected for the Cisco 8102 and Arista 7050CX3 switches. 
```

Comment:
Too many details here. We do not need to mention "IXIA" and "collected for the Cisco 8102 and Arista 7050CX3" here. Just say "test the effect of PFC response times"